vite.config.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    testTimeout: 15000,
```

Comment:
This is not a performance regression. Some tests already took 6 seconds in Jest. But Vitest made it more strict and started failing